src/Aspire.Hosting.Azure.Functions/DurableTask/IResourceWithDashboard.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Azure;
+
+interface IResourceWithDashboard : IResource
```

Comment:
style nit:
```suggestion
internal interface IResourceWithDashboard : IResource
```

Reply:
It's used by both DTS resources, where each implements in a slightly different manner. I've updated the name to be DTS specific.